src/webgl/functions.js

```diff
@@ -0,0 +1,149 @@
+/* eslint-disable */
```

Comment:
This file name is to ambiguous to me. I feel the previous src/webgl/draw.js just works... If lumping functions with various purposes into a single file is what we want, we might call it utility.js, or wrapper.js or similar.

Just for discussion:
I like the way gl.readPixels got wrapped with a readPixels() function, so that deck.gl or any users of luma.gl don't need to call WebGL APIs directly.

But, since one of the major goals of luma.gl is to bridge WebGL's "bind-and-operate-on-global-state" model with more user-friendly "operate-on-object" mode, is it possible to have clear() and readPixels() function part of some objects instead of standalone functions on a global objects?